apps/website/src/components/Card.astro

```diff
@@ -0,0 +1,81 @@
+---
+import React from 'react';
+import ComponentIcon from './ComponentIcon.astro';
+
+export type Props = {
+  url: string;
+  title: React.ReactNode;
+  description?: React.ReactNode;
```

Comment:
this is an astro file, react node is not relevant here. did you mean to use `string` ?

Reply:
I wanted to have more generic component where title could be string or string with icon. But that's probably not going to happen. Need to add other component to home page demo cards.

Reply:
[slots](https://docs.astro.build/en/core-concepts/astro-components/#slots) is the way to do that in astro, but string works for now